src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -373,10 +373,12 @@ class ImageBaseViewer extends BaseViewer {
             if (!this.isPannable && this.isZoomable) {
                 // If the mouse up was not due to panning, and the image is zoomable, then zoom in.
                 this.zoom('in');
+                this.emitMetric('zoom', 'inClick');
```

Comment:
It looks like `emitMetric` takes a string for the first parameter and a `data` object for the second. What event payload does this actually create? Can we discuss the approach in more detail offline?

Reply:
Looks like `data` can be anything, including a string.

Reply:
Yeah, it'll be `JSON.stringify`